src/ViewerComponents/Resource/ResourcesContainer.svelte

```diff
@@ -101,29 +128,58 @@
      * @param data
      */
     function getAmountRange(data) {
-        return `${current_offset * result_per_request + 1}-${current_offset * result_per_request + data['@graph'].length}`;
+        if (data.hasOwnProperty('@graph')) {
+            return `${current_offset * result_per_request + 1}-${current_offset * result_per_request + data['@graph'].length}`;
+        } else {
+            return `${current_offset * result_per_request + 1}`;
+        }
+    }
+
+    /**
+     * Wraps the data in an array if there is one or no result.
+     *
+     * @param data
+     */
+    function wrapData(data) {
+        if (data['@graph']) {
+            return data['@graph'];
+        } else if (data['@id']) {
+            return [data];
+        } else {
+            return [];
+        }
     }
 </script>
 
 {#if requestInfos}
     <div class="container">
-        {#await promise}
-            <Loading/>
+        {#await promise_data}
+            <Loading loading_text="...searching"/>
         {:then data}
             {#if isEmpty(data)}
                 {#if !jsonFile.hasOwnProperty("ShowNone") || jsonFile["ShowNone"]}
                     No data found
                 {/if}
             {:else}
                 {#if !jsonFile.hasOwnProperty("ShowPagination") || jsonFile["ShowPagination"]}
-                    <!-- Pagination Buttons -->
+                    <!-- Pagination buttons -->
                     <button disabled={preventPrevious()} on:click={() => previous()}>&lt;</button>
                     <button disabled={preventNext(data)} on:click={() => next()}>&gt;</button>
-
+                    <!-- Pagination range & amount -->
                     {getAmountRange(data)}
+                    {#await promise_amount then data}
+                        of {data['schema:numberOfItems']}
```

Comment:
also here the "of" should be translated for the different languages

Reply:
See above